extensions/resteasy/deployment/pom.xml

```diff
@@ -40,6 +40,17 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- These are needed to test the JSON representation of the NotFoundExeptionMapper-->
```

Comment:
```suggestion
        <!-- These are needed to test the JSON representation of the NotFoundExceptionMapper-->
```

Reply:
Done.
@gsmet don't worry, it's your job to verify the quality of Quarkus codebase ... and I should have been more careful about this (I really need to re-read twice my code before pushing it) ;)